test/login_external_test.rb

```diff
@@ -42,18 +43,44 @@ def request_session
     end
 
     describe "token is correct" do
-      let(:cookie_data){{ "foo" => "bar" }}
-      let(:token){ JWT.encode(cookie_data, shared_key, "HS256") }
+      let(:jwt_data){{ "foo" => "bar" }}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.